controller/Previewer.php

```diff
@@ -90,19 +109,21 @@ protected function getErrorResponse($e = null)
     protected function getErrorCode($e = null)
     {
         $code = 200;
```

Comment:
this is very weird, the default code of  getErrorCode($e = null) is 200 .. . :/ 😢